src/components/AssetModels/AssetModelsComponent.jsx

```diff
@@ -86,7 +91,8 @@ export class AssetModelsComponent extends React.Component {
 
             <Table.Footer>
               <Table.Row>
-                {!_.isEmpty(this.props.assetModels) && (
+                {!_.isEmpty(this.props.assetModels) &&
+                (
```

Comment:
Nitpick: Move the opening brace to the previous line, the way it was before the change